src/pages/home/subscription/Subscription.component.js

```diff
@@ -30,45 +31,45 @@ const Subscription = () => {
       if (valid) {
         try {
           await api.subscribe.addEmail(email);
-          message.success(lang.emailInput.successMsg);
+          message.success(emailInputLang.successMsg);
         } catch (e) {
           // when API call returns an error
-          message.error(lang.emailInput.errorMsg);
+          message.error(emailInputLang.errorMsg);
         }
       } else {
         // when yup validation fails
-        message.error(lang.emailInput.invalidMsg);
+        message.error(emailInputLang.invalidMsg);
       }
     });
   };
 
   return (
     <Styled.SubscriptionSection>
-      <Styled.Container isMobile={isSmallScreen}>
+      <Styled.Container isSmallScreen={isSmallScreen}>
         <Row>
           <Col xs={24} lg={16}>
-            <Row justify={isSmallScreen && 'center'}>
+            <Row justify={isSmallScreen ? 'center' : undefined}>
```

Comment:
Fixed a non-boolean value warning for the justify property.
<img width="845" alt="Screen Shot 2021-08-24 at 3 08 29 PM" src="https://user-images.githubusercontent.com/123074/130550309-37b4406b-8151-42db-894c-9b8fb22ddea6.png">